app/marketing/views.py

```diff
@@ -407,15 +412,30 @@ def account_settings(request):
 
         if request.POST.get('disconnect', False):
             profile.github_access_token = ''
+            profile = record_form_submission(request, profile, 'account-disconnect')
             profile.email = ''
             profile.save()
             messages.success(request, _('Your account has been disconnected from Github'))
             logout_redirect = redirect(reverse('logout') + '?next=/')
             return logout_redirect
         if request.POST.get('delete', False):
+            # remove profile
             profile.hide_profile = True
+            profile = record_form_submission(request, profile, 'account-delete')
             profile.email = ''
             profile.save()
+
+            # remove email
+            try:
+                client = MailChimp(mc_user=settings.MAILCHIMP_USER, mc_api=settings.MAILCHIMP_API_KEY)
+                result = client.search_members.get(query=es.email)
+                subscriber_hash = result['exact_matches']['members'][0]['id']
+                client.lists.members.delete(
+                    list_id=settings.MAILCHIMP_LIST_ID,
+                    subscriber_hash=subscriber_hash,
+                    )
```

Comment:
When we start running automatic linting/styling checks, the indentation on these parentheses will be automatically moved up to the previous line or the indent will move inward by one with the trailing `,`

Reply:
ok.  maybe i should download a sublime linter.

Reply:
@mbeacom do we follow a strict PEP8 or flake8?

Reply:
@mbeacom as mentioned in [PEP-8](https://www.python.org/dev/peps/pep-0008/#indentation), this might be still correct indentation I think:

> The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character of the last line of list

> or it may be lined up under the first character of the line that starts the multiline construct, as in: